### DISCUSSION TITLE
Use queue path as regular path

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#19146] [Feature requests]
We have a cheat to allow regular path as queue now, so why can't we have queue path as regular path as a cheat?

**Comment by spacek531** [comment]
You can simply do that. No cheats needed.

**Reply by 733737** [reply]
but it won't allow 2 across

**Reply by spacek531** [reply]
Those sprites don't exist for queues, so I don't see why it's a big issue. If someone wants 2-wide or 3-wide paths that look like queues, they can be added as custom path objects with the required sprites added.
